checkov/common/output/report.py

```diff
@@ -175,13 +175,13 @@ def get_test_suites(self) -> List[TestSuite]:
             if record.check_result['result'] == CheckResult.FAILED:
                 if record.file_path and record.file_line_range:
                     test_case.add_failure_info(
-                        f'Resource {record.resource} failed in check {check_name} - {record.file_path}:{record.file_line_range}')
+                        f'Resource {record.resource} failed in check {check_name} - {record.file_path}:{record.file_line_range} - Guideline: {record.guideline}')
```

Comment:
@eurogig very cool. small fix:
- Guideline: {record.guideline}' should be added only if record.guideline!=None 
(same apply for the next change) 

a way to test it: upgrade checkov to latest version, run it without internet connection and see